CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
```

Comment:
Please change the heading to `## [Unreleased]` and then the migration checks will work.

Reply:
Ah, that's the reason. Thank you

Reply:
Actually, AH migration still fails. I can do the same trick as for Kusama BH: https://github.com/polkadot-fellows/runtimes/pull/277/commits/22dc93e6912eb68994c00beaebc3722bfcd631de, but perhaps worth a separate PR

Reply:
Encointer fails for all PRs iiuc